mix.exs

```diff
@@ -43,7 +43,7 @@ defmodule Guardian.Mixfile do
 
   defp deps do
     [{:jose, "~> 1.8"},
-     {:phoenix, "~> 1.2.0", optional: true},
+     {:phoenix, "~> 1.2.0 or ~> 1.2-rc", optional: true},
```

Comment:
> Doing override: true on this dependency did nothing, but adjusting version string made it install.

Can you be more specific of your usage of the `override: true` option?

You would need to use `override: true` on your applications dependency on `phoenix`.

We may be able to loosen our dependency on `phoenix` to `~> 1.2` which should include anything in between `1.2` to `> 2.0`.